CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+When contributing to Precision Converters Firmware please consider the
+following checklist:
+
+- [ ] Copyright header has been added to source and header files
+- [ ] Artistic Style (`astyle`) has been run to lint new code
```

Comment:
I may add an link to the `.gitlab-ci.yml` or simply add an example like:

```bash
$ astyle --options=libraries/no-OS/ci/astyle_config projects/foobar/app/*.c
```

Reply:
I think example would be useful if easy to add here